chain/network/src/network_protocol/network.proto

```diff
@@ -106,6 +106,11 @@ message BlockHeader {
   bytes borsh = 1;
 }
 
+// Wrapper of the borsh-encoded StateResponseInfo.
+message StateResponseInfo {
+  bytes borsh = 1;
```

Comment:
I'm in favor of this if it makes things easier, so that we don't have to define a bunch of stuff in this proto file, but I wonder if going forward, it would be better to fully define new fields instead of borsh-encoding our proto fields. Because this way, we kind of lose a lot of the benefits of protos (but the tradeoff is that it's easier to add stuff of course). Not blocking on this comment, just want to put it out there